### PR TITLE
docs: improve index.md

### PR DESCRIPTION
**Pull request by aarcex3**
# Summary

Updated the documentation to remove the excessive text.
This makes it cleaner and easier to read.

# Closes
None

### CODE REVIEW

**Comment by aarcex3** [comment]
@martinn how you don't mind. It was too much text in my opinion.

File: README.md [hunk 1]
```diff
@@ -102,32 +102,10 @@ It's still early development but so far we have support for:
...
-poetry add quickapiclient
-# Or if you want to use attrs over dataclasses:
-poetry add quickapiclient[attrs]
-# Or if you want to use pydantic over dataclasses:
-poetry add quickapiclient[pydantic]
-# Or if you want to use msgspec over dataclasses:
-poetry add quickapiclient[msgspec]
-# Or if you want to use requests over httpx:
-poetry add quickapiclient[requests]
+pip install quickapiclient[attrs|pydantic|msgspec|requests]
```
Comment (martinn) [inline README.md]:
```suggestion
pip install quickapiclient
# Or with optional extras (choose from the list below)
pip install quickapiclient[attrs,pydantic,msgspec,requests]
# Or if using poetry
poetry add quickapiclient
poetry add quickapiclient[attrs,pydantic,msgspec,requests]
```

**Review by martinn — COMMENTED** [review]
Hey @aarcex3 - thanks again for your contribution, don't mind it at all. :) 

I agree perhaps the current version is too long.

At the same time, I believe `poetry` being one of the most popular package managers at the moment, I want to make it easy for those users to be able to install it. Plus not everybody might be across how to install optional dependencies.

So I've done a suggestion to try and shorten the instructions while still keeping both. Let me know what you think. 

Also open to any other thoughts or ideas, like making it a collapsible link. What do you think?

**Review by martinn — APPROVED** [review]
Apologies I missed the updates on this one. LGTM!

**Pull request merged**